code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -23,7 +23,7 @@
 	. = list("<span class='info'>This is <EM>[!obscure_name ? name : "Unknown"][apparent_species]</EM>!")
 
 	//uniform
-	if(w_uniform && !(obscured & ITEM_SLOT_ICLOTHING))
+	if(w_uniform && !(obscured & ITEM_SLOT_ICLOTHING) && !(w_uniform.item_flags & EXAMINE_SKIP))
```

Comment:
could also be part of a clothing refactor

Reply:
I mean
It's a new flag that's only used for the heretic hoodie, I think I'll keep it